src/pages/dashboard/FilterSearchOptions.tsx

```diff
@@ -36,13 +37,18 @@ const StyledSideBar = styled.div`
   padding-top: 1.5rem;
   padding-left: 1rem;
   background-color: ${Colors.ExploreResourcesPageOptionFiler};
-  min-width: 17rem;
+  width: 17rem;
   display: flex;
   flex-direction: column;
 `;
 
 const StyledAccordionDetails = styled(AccordionDetails)`
   display: flex;
+  flex-direction: column;
```

Comment:
Siden margene på filterboxene er fjernet, har du mulighet til å legge dem inn her?
```suggestion
  flex-direction: column;
  fieldset {
  margin-top: 2rem;
  }
```

Reply:
hehe. Du mener vel egentlig under StyledSideBar ;-)  Det var uansett en bra løsning - så slipper vi spacer

Reply:
fixd

Reply:
:D Mente faktisk inni  StyledAccordionFilterBoxesWrapper, men bra at det kan brukes flere plasser. Når man har mobile-view så blir fieldset'ene veldig nærme hverandre.

Reply:
prøver igjen. Det ble knot pga at man også må ta hensyn til sidebar.